Failed to run phpactor

### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it


<img width="372" alt="Screenshot 2024-12-09 alle 18 21 24" src="https://github.com/user-attachments/assets/002af31e-7ffd-4168-b821-0f2cc40e953a">


```
Language server error: phpactor

server shut down
-- stderr--
env: php: No such file or directory

```

### Environment

```
Zed: v0.165.3 (Zed Preview)
OS: macOS 14.1.1
Memory: 16 GiB
Architecture: x86_64
```

### If applicable, add mockups / screenshots to help explain present your vision of the feature

_No response_

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary>

<!-- Click below this line and paste or drag-and-drop your log-->
```

```
<!-- Click above this line and paste or drag-and-drop your log--></details>

If you'd bothered to look at your logs, if you would have seen:
```
2024-12-16T01:07:52.15149-05:00 [INFO] attempting to start language server "phpactor", path: "/Users/peter/zcode/zed", id: 4
2024-12-16T01:07:52.152176-05:00 [INFO] starting language server process. binary path: "/Users/peter/Library/Application Support/Zed/extensions/work/php/phpactor-2024.11.28.1/phpactor.phar", working directory: "/Users/peter/zcode/zed", args: ["language-server"]
2024-12-16T01:07:52.159293-05:00 [ERROR] cannot read LSP message headers
2024-12-16T01:07:52.159745-05:00 [ERROR] Broken pipe (os error 32)
2024-12-16T01:07:52.159817-05:00 [ERROR] server shut down
2024-12-16T01:07:52.15987-05:00 [ERROR] Failed to start language server "phpactor": oneshot canceled
2024-12-16T01:07:52.159919-05:00 [ERROR] server stderr: "env: php: No such file or directory\n"
```

You haven't install `php`.

```
brew install php
```

Please do better in the future. Take a minute to try and solve your own problems before rushing to open an issue here.

--- a/docs/src/languages/php.md
+++ b/docs/src/languages/php.md
@@ -13,6 +13,18 @@ The PHP extension offers both `phpactor` and `intelephense` language server supp
 
 `phpactor` is enabled by default.
 
+## Phpactor
+
+The Zed PHP Extension can install `phpactor` automatically but requires `php` to installed and available in your path:
+
+```sh
+# brew install php            # macOS
+# sudo apt-get install php    # Debian/Ubuntu
+# yum install php             # CentOS/RHEL
+# pacman -S php               # Arch Linux
+which php
+```
+
 ## Intelephense
 
 [Intelephense](https://intelephense.com/) is a [proprietary](https://github.com/bmewburn/vscode-intelephense/blob/master/LICENSE.txt#L29) language server for PHP operating under a freemium model. Certain features require purchase of a [premium license](https://intelephense.com/). To use these features you must place your [license.txt file](https://intelephense.com/faq.html) at `~/intelephense/licence.txt` inside your home directory.

